pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -443,10 +462,14 @@ func buildMgmtPortListeners(managementPorts model.PortList, managementIP string)
 				ip:       managementIP,
 				port:     mPort.Port,
 				protocol: model.ProtocolTCP,
+				filterChainOpts: []*filterChainOpts{{
+					networkFilters: buildInboundNetworkFilters(instance),
+				}},
 			}
 			l := buildListener(listenerOpts)
-			if err := marshalFilters(l, listenerOpts, buildInboundNetworkFilters(instance), nil); err != nil {
-				log.Warn(err.Error())
+			// TODO: should we call plugins for the admin port listeners too? We do everywhere else we contruct listeners.
```

Comment:
yes, we should. maybe set listener type to ListenerTypeADMIN to make life easy for plugins? i can take this in a followup PR.